src/commands/test/runner.py

```diff
@@ -135,15 +135,21 @@ class ExpectedError:
 
 
 class TestExecutionEnvironment:
-    def __init__(self, is_test_fail_enabled: bool):
+    def __init__(self, is_test_fail_enabled: bool, include_paths):
```

Comment:
It would be nice to have a type here. Sometimes paths are `str`ings, sometimes `Path`s.

Reply:
Ah sorry i missed it